client/src/state-machine.ts

```diff
@@ -74,12 +121,10 @@ type PortfolioData = {
 };
 
 type TimelineData = {
-  timelineBbl: string;
```

Comment:
I'm removing this for now because keeping track of it is a bit of a hassle and it's easier to just set up our state machine so we always reset the timeline/summary states to `noData` whenever our detail address changes.